src/app/pages/OnDemandAudioPage/index.styles.ts

```diff
@@ -0,0 +1,24 @@
+import { css, Theme } from '@emotion/react';
+
+export default {
+  wrapper: ({ mq, spacings }: Theme) =>
+    css({
+      [mq.GROUP_4_MIN_WIDTH]: {
+        paddingTop: `${spacings.TRIPLE}rem`,
+      },
+    }),
+  paragraph: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_1_AND_GROUP_2]: {
+        gridTemplateColumns: 'repeat(4, 1fr)',
+        gridColumnEnd: 'span 4',
+      },
+    }),
+  image: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_1_AND_GROUP_2]: {
```

Comment:
**Note for reviewers**: this is slightly different to the original media query which was https://github.com/bbc/simorgh/blob/99edec329a1360d8bb1cc99591e5da492d68e768/src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx#L65-L66 (which resolves as `@media (min-width: 22.5rem) and (max-width: 37.4375rem)`)

The new value resolves as `@media (min-width: 15rem) and (max-width: 37.4375rem)`

I wanted to use existing media queries, rather than hardcoding the value to 22.5rem

Perhaps we should test this at the smaller breakpoints to see if it's still OK (and get UX 👍 too?)

Reply:
I've decided to stick with the existing layout, as there were significant differences when comparing side-by-side.